lib/utils/dummy_i2c_sensor.hpp

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <sensors/i2c_sensors.hpp>
+
+namespace hyped::utils {
+
+class DummyI2cSensor : public sensors::II2cMuxSensor<std::uint8_t> {
```

Comment:
if this is supposed to be able to imitate any i2c sensor, an `int16_t` might be more suitable here. (_unless we do templating for this as well_  (❛‿❛✿̶̥̥) )

Reply:
this is literally the most trivial sensor i could conjure in order to instantiate a mux. we can make this class less trivial in the future but it's literally just so that i can instantiate the mux template and catch some errors in the generated code.

Reply:
okay fair lmao